### DISCUSSION TITLE
Make database type/vendors configurable

### DISCUSSION THREAD

**Discussion by w0kyj** [directus-labs/helm-chart#61] [Ideas]
Currently the charts and templates are hard coded to a MariaDB. Directus supports multiple vendors and would be great to make the templates and charts the use variables instead to allow for quick setup with any of the supported vendors.

https://docs.directus.io/self-hosted/docker-guide.html#supported-databases
Directus supports the LTS versions of PostgreSQL, MySQL, SQLite, MS SQL Server, MariaDB, CockroachDB, and OracleDB.

**Comment by mykhailo-sindieiev** [comment]
Agree, great idea!

**Comment by fullmetal-fred** [comment]
Echoing this...I used helm to generate manifests, then butchered the manifests to point to an existing postgres instance instead. My use case is a dev instance only at the moment so I ended up only using a secret, configmap, deployment (single replica), and service and stripping out redis.

**Comment by mykhailo-sindieiev** [comment]
Hello @fullmetal-fred, thanks for your feedback. I appreciate it and it gives me insights where move the development of the chart to. For sure, I'll develop support of postgresql and mysql db engines

**Reply by fullmetal-fred** [reply]
@mikesindieiev sounds good. Let me know when you're ready for someone to test and I'll run it in my dev environment. 

Do you want an issue to track Postgres specifically?

**Reply by mykhailo-sindieiev** [reply]
That would be good if you create one for it. 
Most probably, I'll work on that after holiday season

**Reply by fullmetal-fred** [reply]
Have a great holiday!

**Reply by fullmetal-fred** [reply]
See #62

**Reply by mykhailo-sindieiev** [reply]
Thanks! You too

**Comment by mykhailo-sindieiev** [comment]
Release 2.0.0 is live. It supports mysql and postgresql databases. @fullmetal-fred Please let me know you feedback on that

**Comment by mykhailo-sindieiev** [comment]
Correct url to docs - https://directus.io/docs/configuration/database
